src/constants.js

```diff
@@ -26,10 +26,78 @@ const LINK_STRENGHT = 0.3
 const MAX_SCREEN_NODES = 25
 const MARGIN = 37
 const NODE_SIZE = 150
+const personalDetails = [
+  {name: 'name',       optional: false, language: true,  type: 'literal'},
+  {name: 'thumbnail',  optional: true,  language: false, type: 'url'},
+  {name: 'depiction',  optional: true,  language: false, type: 'url'},
+  {name: 'wikiTopic',  optional: false, language: false, type: 'url'},
+  {name: 'dob',        optional: true,  language: false, type: 'literal'},
+  {name: 'dod',        optional: true,  language: false, type: 'literal'},
+]
+const predicates = {
+  influenced:    'dbpedia-owl:influenced',
+  influencedBy: 'dbpedia-owl:influencedBy',
+  depiction: 'foaf:depiction',
+  thumbnail: 'dbpedia-owl:thumbnail',
+  //name: 'foaf:name',
+  name: 'rdfs:label',
+  wikiTopic: 'foaf:isPrimaryTopicOf',
+  occupation: 'dbprop:occupation',
+  dob: 'dbpedia-owl:birthDate',
+  dod: 'dbpedia-owl:deathDate'
+}
+const prefixies = [
```

Comment:
I think that these `prefixies` are more properly tied to the Sparql.js code, rather than constants specific to tsomi.  But that's a decision between two patterns (all constants together v keep internal-only constants alongside the class, so I this isn't s show stopper.